### ISSUE TITLE
Pretraining time of tile encoder

### ISSUE THREAD

**Issue by gabricampanella** [prov-gigapath/prov-gigapath#53]
Dear GigaPath team,

Thank you for your excellent work!
Could you share how long the pre-training of the tile encoder took? In the paper you mention the time for pretraining the slide-level model but not the tile encoder. I have search the open and closed issues but did not find that information either.
It would be greatly appreciated.

**Comment by bug-fixed** [comment]
Same here. In their great paper, it is mentioned as follows:

`We pretrained the model on 1,384,860,229 segmented tiles, treating each tile as one data instance. The base learning rate in DINOv2 pretraining was set to 4 × 10−3. We set the batch size on each GPU device to 12, with a total effective batch size of 384.`

Sounds like the number of GPUs is 32. 

If the time for 10 iterations is 5 seconds, then it needs
        `1384860229/(10*384)*5/3600= 500 hours = 20 days`
for one epoch.

Is this correct? Please correct me if this is wrong. Thanks!
